build.sbt

```diff
@@ -337,6 +337,7 @@ lazy val bootstrap = project in file("target/bootstrap")
 lazy val library = configureAsSubproject(project)
   .settings(generatePropertiesFileSettings)
   .settings(Osgi.settings)
+  .settings(AutomaticModuleName.settings("scala.library"))
```

Comment:
I guess that's the only one (and reflect I guess) that will show up in end-user module files right? "most" users don't care about the compiler module